docs/local_development.md

```diff
@@ -0,0 +1,27 @@
+# Recommended Local Development and Debugging Workflow
+
+## Overview
+
+We recommend 2 modes of running the broker for development work.
+
+1. Build a local image and deploy from image
+2. Build the broker executable and run it locally connected to a local oc cluster up from [fusor/catasb 'dev' branch](https://github.com/fusor/catasb/tree/dev)
+
+## Build a local image and deploy from image
+
+1. Build the Ansible Service Broker executable with the command: ```make build```
+2. Edit the Makefile to reflect the BROKER_IMAGE_NAME and TAG you want to use for local development
+3. Build a development image:  ```make build-image```
```

Comment:
merge step 2 and 3. `make build-image BROKER_IMAGE_NAME=asb-dev TAG=johnny5`. Editing the `Makefile` is a bad idea.